src/c++/vdo/base/dm-vdo-target.c

```diff
@@ -3084,7 +3097,9 @@ static int __init vdo_init(void)
 #ifdef __KERNEL__
 	/* Memory tracking must be initialized first for accurate accounting. */
 	vdo_memory_init();
+#ifdef VDO_INTERNAL
 	uds_init_sysfs();
+#endif
 
 	vdo_initialize_thread_device_registry();
```

Comment:
As long as this needs updating, I'd get rid of the blank line, too. The UDS/VDO distinction it was trying to highlight is effectively gone.